kafka/4.8/modules/ROOT/pages/kafka-connector-configure-confluent-example.adoc

```diff
@@ -0,0 +1,180 @@
+= Integrate Confluent Schema Registry with Kafka Connector - Example 
+
+The following example shows how to configure the Confluent Schema Registry using Kafka Connector. 
+
+Imagine you are developing a Mule 4 app that processes streaming data from various sources. To ensure the data conforms to defined schemas and to manage schema evolution effectively, you integrate Confluent Schema Registry with your Kafka Connector. This setup is crucial for developers who need to enforce data integrity and compatibility across different versions of data producers and consumers.
+
+The configuration consits of two parts:
+
+* Configure the Confluent Schema Registry in the Kakfa Connector connection using the *Additional Properties* field  (either for the *Consumer* or *Producer* operation).
+* Configure the Kafka Avro dependency in the POM file of the Mule app. 
+
+== Configure Kafka Connection
```

Comment:
Is this the same as the first part mentioned before? If so, I'd update the title to be more similar: 

== Configure the Confluent Schema Registry in the Kakfa Connector connection